tests/cloud_playbooks/roles/packet-ci/templates/inventory.j2

```diff
@@ -107,10 +126,15 @@ instance-2 etcd_member_name=etcd3
 {% endif %}
 
 [k8s-cluster:children]
-kube-node
+kube_node
+kube-master
+calico_rr
+
+[k8s_cluster:children]
+kube_node
 kube-master
-calico-rr
+calico_rr
```

Comment:
you have a weird mix of - and _, does the following make more sense ?
```
[k8s-cluster:children]
kube-node
kube-master
calico-rr

[k8s_cluster:children]
kube_node
kube_control_plane
calico_rr
```

Reply:
Indeed, it looks cleaner this way. I modified in the latest push.